gdctx/global.go

```diff
@@ -71,7 +71,12 @@ func doInit() {
 
 	Rest = rest.New()
 
-	initStore()
+	if Restart == true {
```

Comment:
Couldn't this just be `InitStore(!Restart)`. Add a comment clarifying what's being done above this, and everyones happy.